python/skupper_router/management/skrouter.json

```diff
@@ -368,6 +368,143 @@
             "attributes": {}
         },
 
+        "routerStats": {
+            "description": "Overall router statistics.",
+            "extends": "operationalEntity",
+            "attributes": {
+                "id": {
+                    "description": "Router's unique identity. The router will fail to start without id.",
+                    "type": "string",
+                    "required": false,
+                    "create": true
+                },
+                "version": {
+                    "type": "string",
+                    "description": "Software Version"
+                },
+                "addrCount": {
+                    "type": "integer",
+                    "description": "Number of addresses known to the router.",
+                    "graph": true
+                },
+                "linkCount": {
+                    "type": "integer",
+                    "description": "Number of links attached to the router node.",
+                    "graph": true
+                },
+                "nodeCount": {
+                    "type": "integer",
+                    "description": "Number of known peer router nodes.",
+                    "graph": true
+                },
+                "linkRouteCount": {
```

Comment:
Can we drop linkRouteCount since this is a new record?

Reply:
Removed linkRouteCount